linkml-schema/kbmodel.yaml

```diff
@@ -96,34 +78,222 @@ classes:
         inlined_as_list: true
         range: genome assembly
 
+  process output:
+    description: >-
+      The output of a process.
+    slots:
+      - label
```

Comment:
should we add a description in the `slot_usage` that this is always donor's label